shared/main.lua

```diff
@@ -9,12 +9,12 @@ for i = 97, 122 do StringCharset[#StringCharset + 1] = string.char(i) end
 
 function QBShared.RandomStr(length)
     if length <= 0 then return '' end
-    return QBShared.RandomStr(length - 1) .. StringCharset[math.random(1, #StringCharset)]
+    return ('%s%s'):format(QBShared.RandomStr(length - 1), StringCharset[math.random(1, #StringCharset)])
```

Comment:
```suggestion
    return QBShared.RandomStr(length - 1) .. StringCharset[math.random(1, #StringCharset)]
```
Maybe change this back, it's less efficient and also doesn't really improve readability.

Reply:
Not about it being faster, more about general practice, format it rather than concatenate it.

Reply:
Yeah in this case it should be the other way around, in cases like logs it'd make more sense to use format, although I didn't change those as it's a pain right now to convert them all